source/patterns/@aws-solutions-constructs/aws-apigateway-s3/README.md

```diff
@@ -0,0 +1,94 @@
+# aws-apigateway-s3 module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_apigateway_s3`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-apigateway-s3`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.apigateways3`|
+
+## Overview
+This AWS Solutions Construct implements an Amazon API Gateway REST API connected to Amazon S3 bucket(s).
+
+Here is a minimal deployable pattern definition in Typescript:
+
+``` typescript
+import { ApiGatewayToS3Props, ApiGatewayToS3 } from "@aws-solutions-constructs/aws-apigateway-s3";
+
+new ApiGatewayToS3(this, 'test-api-gateway-s3-default', {
+    apiGatewayProps: {
```

Comment:
Minimal means leave stuff like this out. That said, I may have an issue with the other constructs along these lines. They list the minimal deployment as:

```
new ApiGatewayToSqs(this, 'ApiGatewayToSqsPattern', {});
```

This is great as it is absolutely minimal, but if the allow parameters default to false then this deploys a construct that doesn't do anything.  See below about explicitly stating the default values in the docs - if they are false we'll need to turn at least 1 on.

Reply:
Noted. Will be updated with at least one allow parameter turned on.